source/background.ts

```diff
@@ -42,3 +42,10 @@ browser.runtime.onInstalled.addListener(async ({reason}) => {
 
 // GitHub Enterprise support
 addDomainPermissionToggle();
+
+// `background` fetch required to avoid avoid CORB introduced in Chrome 73 https://chromestatus.com/feature/5629709824032768
+browser.runtime.onMessage.addListener(({action, payload}): Promise<Response> | void => {
```

Comment:
Does this cause trouble in the polyfill?
```suggestion
browser.runtime.onMessage.addListener(async ({action, payload}): Promise<Response | void> => {
```

Reply:
I just read the docs. Let's add this as a comment
```suggestion
// Don’t turn this into an `async` function https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessage#addListener_syntax
browser.runtime.onMessage.addListener(({action, payload}): Promise<Response> | void => {
```